acceptance/tests/facts/windows_os.rb

```diff
@@ -0,0 +1,21 @@
+test_name "Test facter os reports correct os.windows.system32" do
```

Comment:
We should assert that the output is also correct before activating the RDS service

Reply:
On 5.5.x branch was broken regardless of Remote Desktop feature installation, on 6.4.x was broken after installation. With this commit, facter will be consistent all the time.